packages/ui/v2/modules/event-types/DestinationCalendarSelector.tsx

```diff
@@ -4,7 +4,7 @@ import React, { useEffect, useState } from "react";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { DestinationCalendar } from "@calcom/prisma/client";
 import { trpc } from "@calcom/trpc/react";
-import Select from "@calcom/ui/v2/core/form/Select";
```

Comment:
Why have we moved it to smallcase? Actual files still seems to be Capital case. This might break on non unix systems

Reply:
It's a directory now with an `index.ts`, a `Select.tsx` (PascalCase) and `components.tsx`. The directory itself is lowercase since it isn't a component file? I think that makes most sense, right?

Reply:
Oooooops. Didn't notice that. Yeah totally makes sense.